ext/ddtrace.stub.php

```diff
@@ -818,6 +818,11 @@ function dd_trace_serialize_closed_spans(): array {}
      */
     function dd_trace_peek_span_id(): string {}
 
+    /**
+     * Force close all spans and force-send finished traces to the agent
```

Comment:
This drops or closes the spans depending on `datadog.autofinish_spans`, right? Can we please add a note about that in the docblock here?